### PR TITLE
Timer: Changed query_raw to query_lc

### PR DESCRIPTION
**Pull request by shivam99aa**
## Description of new Instant Answer, or changes
This change will replace from using raw string to more normalized string.

## Related Issues and Discussions
Discussed in pull request [4134](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4134)

## People to notify
@moollaza 

Instant Answer Page:  https://duck.co/ia/view/timer

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by shivam99aa** [comment]
@moollaza I have only changed the regex which I submitted in pull request [4134](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4134). 

I updated other regexes too on my test environment but did not commit those as I am not sure if you want those to be edited or not.

**Comment by shivam99aa** [comment]
@moollaza Can you please review this and update if i should update other regexes or not?

**Comment by mattr555** [comment]
> I updated other regexes too on my test environment but did not commit those as I am not sure if you want those to be edited or not.

If the tests pass, please do commit those and I'll take a look.

File: lib/DDG/Goodie/Timer.pm [hunk 1]
```diff
@@ -87,7 +87,7 @@ sub build_result {
 
 handle remainder => sub {
     my $qry = $_;
-    my $raw = lc($req->query_raw);
+    my $raw = lc($req->query_lc);
```
Comment (mattr555) [inline lib/DDG/Goodie/Timer.pm]: You can remove the call to `lc` here.

File: lib/DDG/Goodie/Timer.pm [hunk 2]
```diff
@@ -109,7 +109,7 @@ handle remainder => sub {
 
     # When ambigous words are present in triggers then it should not
     # invoke Timer IA.
-    if($raw =~ /^\s*$ambTrgx\s*$/i){
+    if($raw =~ /^$ambTrgx$/i){
```
Comment (mattr555) [inline lib/DDG/Goodie/Timer.pm]: This regex (and all the others) can also have the `i` flag removed here for a small optimization. 😀

Comment (pjhampton) [inline lib/DDG/Goodie/Timer.pm]: quick wins are the best wins 👍

**Comment by shivam99aa** [comment]
@mattr555 I have updated the code with the changes you asked for. Also, issue #4188 is also solved in this pull request.
Since that issue arises due to me missing brackets in my previous PR, I do not feel that a new PR is needed for that.

**Comment by shivam99aa** [comment]
Sorry to bug you guys again but can anyone check this and merge this pull request. 
@moollaza @mattr555 @pjhampton ^^

**Comment by mintsoft** [comment]
@shivam99aa this looks fine; are there any new query patterns that should be triggered from this? If so can you make sure you add a test for it

**Comment by shivam99aa** [comment]
@mintsoft This is not suppose to trigger any new query patterns but this also contain a bug fix for issue #4188 and I have added one test case for that to be on safe side.

File: lib/DDG/Goodie/Timer.pm [hunk 3]
```diff
@@ -109,7 +109,7 @@ handle remainder => sub {
 
     # When ambigous words are present in triggers then it should not
     # invoke Timer IA.
-    if($raw =~ /^\s*$ambTrgx\s*$/i){
+    if($raw =~ /^($ambTrgx)$/){
```
Comment (mattr555) [inline lib/DDG/Goodie/Timer.pm]: Good spot, this looks good to me 👍

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**